libkbfs/init.go

```diff
@@ -235,7 +235,8 @@ func AddFlags(flags *flag.FlagSet, ctx Context) *InitParams {
 	flags.IntVar((*int)(&params.BGFlushDirOpBatchSize), "sync-batch-size",
 		int(defaultParams.BGFlushDirOpBatchSize),
 		"The number of unflushed directory operations in a TLF that will "+
-			"trigger an immediate data sync.")
+			"trigger an immediate data sync.  EXPERIMENTAL, only "+
+			"available for Keybase admins")
```

Comment:
Perhaps mentioned here in the help doc that the parameter gets reverted to default if logged out and logged back in again?